src/PrestaShopBundle/Install/Install.php

```diff
@@ -1006,15 +1006,20 @@ public function postInstall(): bool
             return $module->get('name');
         }, iterator_to_array($moduleCollection));
 
-        return $this->executeAction(
+        if (!$this->executeAction(
             $modules,
             'postInstall',
             $this->translator->trans(
                 'Cannot execute post install on module "%module%"',
                 ['%module%' => '%module%'],
                 'Install'
             )
-        );
+        )) {
+            return false;
+        }
+
+        // Remove the install lock file
```

Comment:
just a note - Instead of such comments which doesn't give absolutely any benefit (because the code is verbose enough and talks for its self), I would rather appreciate the explanation of what is that lock file purpose above the constant itself